libdebug/commlink/pipe_manager.py

```diff
@@ -155,42 +205,50 @@ def _recvonceuntil(
             drop (bool, optional): drop the delimiter. Defaults to False.
             timeout (float, optional): timeout in seconds. Defaults to timeout_default.
             stderr (bool, optional): receive from stderr. Defaults to False.
+            optional (bool, optional): if ignore the until component of the receive when the child process is not running. Defaults to False.
```

Comment:
What?

Reply:
what what? the name? the description? I lost the capability to read in your mind :(

Reply:
The everything, I don't understand what this is supposed to mean

Reply:
Love the dissing. Remember this ends up in the documentation generated from Pydoc, so be as clear as you can even if it ends up being a little lengthier.

Reply:
I suggest changing the description to "Whether to ignore the wait for the received input if the command is executed when the process is stopped"